hivemind_core/protocol.py

```
@@ -228,6 +257,38 @@ def __post_init__(self):
                                                                   agent_protocol=self.agent_protocol)
         else:
             self.binary_data_protocol.hm_protocol = self
+        if self.policy_chain is None:
+            from hivemind_core.policy import MessageTypeACLPolicy, DenyAllPolicy
+            cfg = get_server_config()
+            try:
+                chain = PolicyChain.from_config(cfg, hm_protocol=self)
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)